examples/type-conversion/type_conversion.bal

```diff
@@ -24,64 +27,59 @@ function convertEmployeeToPerson(Employee emp) {
     }
 }
 
-// This function attempts to convert an `any` constrained map to an `anydata` record `Person`.
-// The conversion would return an error if non-`anydata` or an incompatible value is found.
-function convertAnyMapToPerson(map<any> m) {
-    // Attempts to create a new value of the type `Person` from the `map<any>` typed `m` value without changing
+// This function attempts to convert an `anydata` constrained map to an `anydata` record `Person`.
+// The conversion would return an error if an incompatible value is found.
+function convertAnydataMapToPerson(map<anydata> m) {
+    // Attempts to create a new value of the type `Person` from the `map<anydata>` typed `m` value without changing
     // the inherent type of `m`.
-    Person|error res = Person.convert(m);
+    Person|error res = Person.constructFrom(m);
     if (res is Person) {
         // If the conversion is successful, this prints the `name` field.
-        io:println("map<any> to Person, name: ", res["name"]);
+        io:println("map<anydata> to Person, name: ", res["name"]);
     } else {
-        io:println("Error occurred on conversion");
+        io:println("Error occurred on conversion: ", res.detail());
     }
 }
 
-// This function converts simple basic types using the `.convert()` built-in method.
-function convertSimpleBasicTypes() {
+// This function creates numeric values from string values.
+function convertStringsToNumericTypes() {
     string s1 = "45";
     string s2 = "abc";
-    string s3 = "true";
-    float f = 10.2;
-    any a = 3.14;
+    string s3 = "12.3";
+    string s4 = "8";
 
     // The `string` to `int` conversion is unsafe since the `string` value may not be convertible to `int`.
```

Comment:
Instead of convert, shall we remove this and change the next sentence to say
 ```ballerina
    // The `fromString()` method in the ballerina/lang.int module returns the integer value represented by a given string
    // if there is a valid representation, else returns an `error`.
```

Let's change all the comments similarly.